Makefile

```diff
@@ -14,6 +14,7 @@ clean: ## Cleans compiled
 	@cargo clean
 
 clean-node: ## Cleans local dbs needed for sequencer and nodes
+	sudo rm -rf data/postgres
```

Comment:
We should also ignore data folder in hive/dockerignore

Reply:
`.dockerignore` file added.